dasharo-security/secure-boot.robot

```diff
@@ -9,19 +9,22 @@ Library             RequestsLibrary
 # TODO: maybe have a single file to include if we need to include the same
 # stuff in all test cases
 Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../pikvm-rest-api/pikvm_comm.robot
 Resource            ../rtectrl-rest-api/rtectrl.robot
 Resource            ../variables.robot
 Resource            ../keywords.robot
 Resource            ../keys.robot
+Resource            ../lib/secure-boot-lib.robot
```

Comment:
This can go into keywords.robot

Reply:
Resolved in [21a0c1f](https://github.com/Dasharo/open-source-firmware-validation/pull/17/commits/21a0c1f4bc636ec5ee87d9cd923764a084c70656)